src/components/Login.tsx

```diff
@@ -105,77 +105,91 @@ function Login() {
-                                    show="block"
-                                    label="Select Account"
-                                    type="submit"
-                                />
-                            </Form>
-                        ),
-                    })
-                } else if (response.status === 401) {
+                                    onSubmit={() => {
+                                        showModal({ active: false })
+                                    }}
+                                >
+                                    {accounts.map((element: string) => (
+                                        <label key={element}>
+                                            <input
+                                                type="radio"
+                                                id={element}
+                                                value={element}
+                                                name="radio-group"
+                                                defaultChecked={false}
+                                                onChange={(event: any) => {
+                                                    setAccountName(
+                                                        event.target.value,
+                                                    )
+                                                }}
+                                            />
+                                            &nbsp;{element}
+                                        </label>
+                                    ))}
+                                    <Button
+                                        size="large"
+                                        kind="primary"
+                                        show="block"
+                                        label="Select Account"
+                                        type="submit"
+                                    />
+                                </Form>
+                            ),
+                        })
+                    } else if (response.status === 401) {
+                        showAlert({
+                            message: 'Invalid credentials',
+                            variant: 'error',
+                        })
+                    } else {
+                        showAlert({
+                            message: 'Login failed.',
+                            variant: 'error',
+                        })
+                        throw new Error('Login failed')
+                    }
+                } else {
                     showAlert({
-                        message: 'Invalid credentials',
+                        message: 'Unable to fetch response from server.',
                         variant: 'error',
                     })
-                } else {
-                    throw new Error('Login failed')
                 }
                 return response
```

Comment:
this can be removed ;)

Reply:
Sure!

Reply:
Hey @nimishbongale, I meant we should remove `return response`. In case of an error, we would return `undefined` from this block, which would not be handled by this function. Would you agree that we should handle the expectations here or throw a well-defined expectation from the restApiWrapper functions?

Reply:
Ah! Alright! Makes sense. Will do the needful 😄